client/utils/collections/api.ts

```diff
@@ -1,6 +1,18 @@
 import { apiFetch } from 'client/utils/apiFetch';
 
 const collectionPubsRoot = '/api/collectionPubs';
+const collection = '/api/collections';
+
+export const createTagCollection = ({ title, communityId, isPublic = true }) =>
```

Comment:
I'd set `isPublic = false` here (realizing my previous comment about this was a little hard to interpret)